rust/src/lib.rs

```diff
@@ -755,7 +753,7 @@ pub extern "C" fn zcashlc_get_verified_transparent_balance(
         let addr = unsafe { CStr::from_ptr(address).to_str()? };
         let taddr = TransparentAddress::decode(&network, &addr).unwrap();
         let amount = (&db_data)
-            .get_target_and_anchor_heights()
+            .get_target_and_anchor_heights(ANCHOR_OFFSET)
```

Comment:
we should double check that this aligns with the wallet balance that `shield_funds` uses. There's currently a Bug of `shield_funds` requiring 20 confirmations instead of 10.